### ISSUE TITLE
DIB lab code review session, Mon 9/12 @ 3pm

### ISSUE THREAD

**Issue by johnsolk** [ljcohen/MMETSP#5]
I would like to share this repo with people but the code needs polishing and work. I'm not sure exactly how to tackle this issue - [pydoit](http://www.camillescott.org/2016/01/20/pydoit-workshop/)? - and welcome any comments and suggestions. 

We'll meet in lab on Mon, 9/12 at 3pm. Please feel free to comment here too.

**Comment by ctb** [comment]
Misc comments from an initial review:

* the `Trinity.pep.longest` in `mmetsp_trinity_finished` should probably not be checked into the repo. They make it very, very big.  If they must be checked in, they should be gzipped.
* likewise, plain text files in counts/ should be compressed. These seem small enough to remain in the repo.
* what are these files and do they need to be there?

```
MMETSP_SRA_Run_Info_subset2.csv
MMETSP_SRA_Run_Info_subset_a.csv
MMETSP_SRA_Run_Info_subset_b.csv
MMETSP_SRA_Run_Info_subset_c.csv
MMETSP_SRA_Run_Info_subset_d.csv
ScreenShot_2015-10-04.png
Untitled.ipynb
Untitled1.ipynb
Trinity.fasta
batch1.mmetsp.OGcounts.csv
batch1.mmetsp.OGcounts.filtered.csv
clustered.c3.s9.SRR_batch1.fa
```

* what are these files and should they be documented or removed?

```
MMETSP_list
run.sh
SraRunInfo.csv
```

* are all these files data files? Put them in data/ & document what they are:

```
MMETSP_SRA_Run_Info_subset2.csv
MMETSP_SRA_Run_Info_subset_a.csv
MMETSP_SRA_Run_Info_subset_b.csv
MMETSP_SRA_Run_Info_subset_c.csv
MMETSP_SRA_Run_Info_subset_d.csv
MMETSP_reverse_transrate_data.csv
MMETSP_transrate_data.csv
busco_scores.csv
transrate_reference_scores_cds.csv
transrate_reference_scores_nt.csv
transrate_reverse_scores_cds.csv
transrate_reverse_scores_nt.csv
transrate_scores.csv
```

* put these files under notebooks/, I think

```
Extract_Clusters.ipynb
MMETSP_Trinity_transrate_busco_scores.ipynb
MMETSP_reverse_transrate.ipynb
MMETSP_transrate.ipynb
basic_plots.ipynb
```

**Comment by ctb** [comment]
The guiding rationale is that a clean checkout of this repo should be fast (~10-20 seconds on a reasonable internet connection), and a directory listing should be (relatively) clean, with most collections of many files placed within subdirectories.

**Comment by ctb** [comment]
The master branch at https://github.com/ljcohen/MMETSP is the branch to look at!

**Comment by ctb** [comment]
@ljcohen is the list of scripts in the README.md (under "Automated scripts running the protocols") still accurate and up to date?

**Comment by johnsolk** [comment]
Yes, the list of 5 files are the main scripts that work right now:
1. `getdata.py`, download data from NCBI and organize into individual directories for each sample/accession ID
2. `trim_qc.py`, trim reads for quality, interleave reads
3. `diginorm_mmetsp.py`, normalize-by-median and filter-abund from khmer, rename, combined orphans,
4. `assembly.py`, runs Trinity de novo transcriptome assembly software
5. `salmon.py`

Added a list of other scripts to `README.md`. Want to to incorporate `dammit.py`. Haven't run this on all samples yet.

**Comment by jessicalumian** [comment]
Google doc with comments: https://docs.google.com/document/d/1vUs9gpFQfdXroqJ04JTzb4Vp3IHxXu4GAGOhrZXXjgs/edit
